presto-docs/src/main/sphinx/connector/hana.rst

```diff
@@ -0,0 +1,101 @@
+====================
+HANA Connector
+====================
+
+The HANA connector allows querying and creating tables in an
+external HANA database. This can be used to join data between
+different systems like HANA and Hive, or between two different
+HANA instances.
```

Comment:
I'm a bit confused about the support for creating tables. It's mentioned here that creating tables is supported, but then it says in the limitations section below that it isn't. We should probably clarify that and then add some tests as well if applicable

Reply:
Thanks @kiersten-stokes. I just removed the redundant codes according to your comments. I have tested and clarified that `CREATE TABLE` and `DROPT TABLE` statements are supported.  But it is difficult to add the tests for HANA since it is very heavy to create an embedded HANA server. The only way to create an embedded HANA server is to build with docker image [SAP HANA, express edition]( https://hub.docker.com/_/sap-hana-express-edition). Unfortunately, it is no longer maintained (it takes a large amount of RAM to create even if it still works.).